airflow/providers/amazon/aws/operators/datasync.py

```diff
@@ -331,16 +372,52 @@ def _update_datasync_task(self):
         self.log.info("Updated TaskArn %s", self.task_arn)
         return self.task_arn
 
-    def _execute_datasync_task(self):
-        """Create and monitor an AWSDataSync TaskExecution for a Task."""
-        hook = self.get_hook()
+    def _wait_get_status_before_start(
+            self,
+            max_iterations=12 * 180):  # wait_interval_seconds*12*180=180 minutes by default
```

Comment:
```suggestion
            max_iterations : int = 12 * 180) -> str:
```
You already have that comment in the docs below. Even better it would be to add `:param` and add the comment there.